test/parallel/taskCompare/elliot/empty-omp-parallel-for.skipif

```diff
@@ -0,0 +1,2 @@
+# cce optimizes away the loop entirely resulting in a no-op test
+CHPL_TARGET_COMPILER=cray-prgenv-cray
```

Comment:
Yeah, I thought about that. works with gnu and intel. And we don't perf test with pgi or clang